src/main/java/games/strategy/engine/framework/startup/ui/LocalSetupPanel.java

```diff
@@ -146,15 +146,6 @@ public void update(final Observable o, final Object arg) {
     layoutComponents();
   }
 
-  public String getPlayerType(final String playerName) {
```

Comment:
The last call to this method appears to have been removed in 6bf3caf (when this class was previously named `LocalPlayerSelectionPanel`).